vignettes/articles/sliding.Rmd

```diff
@@ -68,8 +62,7 @@ versions for the less up-to-date input archive.
 ```{r grab-epi-data}
 theme_set(theme_bw())
 
-y <- readRDS("all_states_covidcast_signals.rds")
-
+y <- readRDS(here::here("vignettes/articles/all_states_covidcast_signals.rds"))
```

Comment:
It doesn't need this change. Vignettes are built with direct access to the directory in which they live. 

If you want this data directly available to the user, it would need to go in `inst/*`

Reply:
I couldn't execute this code in the REPL without this. I could remove this and find a better solution when we actually do an overhaul of the vignettes.

Reply:
That's correct. It won't run at the REPL but will when the vignette is rendered.